service/src/test/java/bio/terra/workspace/service/workspace/ApplicationUnitTest.java

```diff
@@ -249,4 +249,13 @@ private WsmApplication configValidationSuccess(App testApp) {
     assertEquals(wsmApp.getState().name(), GOOD_STATE);
     return wsmApp;
   }
+
+  private boolean findEqualApp(List<WsmApplication> wsmApps, WsmApplication wsmApp) {
```

Comment:
what's wrong with just `List#contains()`?

Reply:
I previously changed that to use the `hasItem` assert, but forgot to delete the function.